src/contents/all/components/app/App.tsx

```diff
@@ -69,11 +69,19 @@ const App = () => {
 
 
   const updateLogStatus = target => {
-
-    const newLog = { ...log, status: target.key }
-    setLog(newLog)
-    setMessage(statusMessage[target.key])
-    chrome.runtime.sendMessage({ message: "log_update", log: newLog })
+    if (target.key == "delete") {
+      const delLog = { ...log, status: target.key }
+      chrome.runtime.sendMessage({ message: "delete_log", log: delLog })
```

Comment:
Then here you can simplify this. No need to create a new log object and just pass in the existing log id to the `delete_log` message:

```suggestion
	      chrome.runtime.sendMessage({ message: "delete_log", id: log.id })
```